docs/source/install.rst

```diff
@@ -0,0 +1,71 @@
+Installation
+============
+
+Project Koku is divided into logical components. To achieve a fully functional
+application, it requires the deployment of multiple micro-applications.
+
+The primary applications for Project Koku are:
+
+* Koku API (Reporting and Query API service)
+* Koku UI (Front-end Web UI)
+* Masu (Data ingestion service)
+
+For development and testing, Project Koku also includes:
+
+* Nise (Test data generator)
+* Hansei (Test automation framework)
+
+This guide will focus on deploying Project Koku into an existing `OpenShift <https://www.okd.io/>`_ cluster.
+
+Deploying the Koku API
+----------------------
+
+The Koku application contains two components - a web service and database.
+
+A basic deployment configuration is contained within the application's `openshift template file <https://github.com/project-koku/koku/blob/master/openshift/koku-template.yaml>`__. This template should be acceptable for most use cases. It provides parameterized values for most configuration options.
+
+To deploy the Koku API application using the provided templates, you can use
+the provided ``Makefile``:
+
+    ``make oc-create-all``
+
+To deploy individual components, there are also ``make`` commands provided for your convenience:
+
+    Deploy the API web application: ``make oc-create-koku``
+    Deploy the PostgreSQL database: ``make oc-create-db``
+
+Deploying the Koku UI
+----------------------
+
+The Koku-UI application is the web-based frontend for Project Koku. It is built
```

Comment:
I feel like it would be better to have a pointer to the Koku UI project "Deployment" section so that repo can stay internally up-to-date, vs. having to make changes to Koku.